v2alpha1/api/swagger/components/schemas/MachineInventory.yaml

```diff
@@ -3,15 +3,15 @@ description: >-
   An entry describing the number of machines of a particular machine size at a
   specific location
 properties:
-  LocationID:
+  locationID:
     type: string
     format: uuid
     description: Unique ID of the data center location
-  SizeID:
+  sizeID:
     type: string
     format: uuid
-    description: ID of a MachineSize as listed in the MachineSizes array
-  Number:
+    description: ID of a MachineSize as listed in the 'machineSizes' array
```

Comment:
Since we're starting over w/ v2, can we converge to GLC terminology and change all MachineSize to InstanceTypes, and all Projects to ComputeGroups?  I had already asked Denis to do the same.

Reply:
Nevermind.  Missed the comment about the MachineSize/Project changes in a separate PR.